mhs/inbound/inbound/request/handler.py

```diff
@@ -46,6 +47,8 @@ def initialize(self, workflows: Dict[str, workflow.CommonWorkflow],
     @time_request
     async def post(self):
         logger.info('Inbound POST received: {request}', fparams={'request': self.request})
+        if logger.isEnabledFor(logging.DEBUG):
```

Comment:
Need this check because `decode()` fails on mock and it's easier to add this check there than modify tons of tests

Reply:
You can consider: https://stackoverflow.com/questions/1835756/using-try-vs-if-in-python